nos/cli/serve.py

```diff
@@ -99,6 +102,7 @@ def _serve_up(
     http_port: int = typer.Option(8000, "--http-port", help="HTTP port to use.", show_default=True),
     http_workers: int = typer.Option(1, "--http-workers", help="HTTP max workers.", show_default=True),
     logging_level: str = typer.Option("INFO", "--logging-level", help="Logging level.", show_default=True),
+    nos_home: str = typer.Option("~/.nosd", "--nos_home", help="NOS home override.", show_default=True),
```

Comment:
Use `home_directory: str` and `--home-directory` with hyphens for CLI arguments.

nit `help="Override the NOS_HOME variable with a custom location"`